mars/worker/storage/sharedstore.py

```diff
@@ -14,6 +14,13 @@
 
 import logging
 
+import pyarrow
+from pyarrow import plasma
+try:
+    from pyarrow.plasma import PlasmaObjectNonexistent, PlasmaStoreFull
+except ImportError:
+    from pyarrow.lib import PlasmaObjectNonexistent, PlasmaStoreFull
+
```

Comment:
Errors caused by pyarrow 0.15.0 has already been fixed in #771.

Reply:
Rebased.